exercises/raindrops/example.py

```diff
@@ -1,10 +1,16 @@
-DROPS = ((3, 'Pling'), (5, 'Plang'), (7, 'Plong'))
-
-
-def convert(number):
+def convert(number: int) -> str:
```

Comment:
As awesome as type annotations are, they are not valid syntax in Python 2. Once we drop Python 2 (sometime in the next few months), this feature will be fine, but Travis fails when it's used right now.

Reply:
Grr. Fixed.